tox.ini

```diff
@@ -29,9 +29,13 @@ deps =
     py{27}: enum34
 
     # Python3.5+ only deps
-    py{35,36,37,38}: aiohttp >= 3.0.0
+    py{35,36,37,38}: aiohttp >= 3.3.1
     py{35,36,37,38}: pytest-aiohttp
-    py{35,36,37,38}: aiobotocore >= 0.10.0
+    py{35,36,37,38}: botocore >= 1.15.3,<1.15.16
+    # aiobotocore 0.12.0 is not available for python 3.5
+    py{35}: aiobotocore >= 0.10.0
```

Comment:
I think it will look for latest version so you can add this case with other cases. py{35,36,37,38} >= 0.12.0

Reply:
actually we may not even need this case at all. having py{35,36,37,38} >= 0.10.0 should pull in whatever latest version is supported for each runtime.

Reply:
Yeah. That's what I am suggesting. we can keep 0.10.0